src/routes/(admin)/tests/test-[type]/[id]/QuestionDialog.svelte

```
@@ -0,0 +1,82 @@
+	const data = $derived.by(() =>
+		questions
+			.filter((question) => {
+				// Tag filter (if any tags are selected)
+				if (tags.length > 0) {
+					const hasMatchingTag = question.tags?.some((tag: any) => tags.includes(String(tag.id)));
+					if (!hasMatchingTag) return false;
+				}
+
+				// State filter (if any states are selected)
+				if (states.length > 0) {
+					const hasMatchingState = question.locations?.some((location: any) =>
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)